debs/Debian/amd64/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN echo "deb http://archive.debian.org/debian/ wheezy contrib main non-free" >
     echo "deb-src http://archive.debian.org/debian/ wheezy contrib main non-free" >> /etc/apt/sources.list && \
     apt-get update && apt-get install -y apt-utils && \
     apt-get install -y --force-yes \
-    curl gcc make cmake sudo expect gnupg perl-base=5.14.2-21+deb7u3 perl \
+    curl gcc make wget sudo expect gnupg perl-base=5.14.2-21+deb7u3 perl \
```

Comment:
```suggestion
    curl gcc make sudo expect gnupg perl-base=5.14.2-21+deb7u3 perl \
```

Reply:
It's required
`./contrib/download_prerequisites: 37: ./contrib/download_prerequisites: wget: not found`